CrossMod/SortingOption.cs

```diff
@@ -11,13 +11,10 @@
 using Terraria.ModLoader;
 
 namespace MagicStorage.CrossMod {
-	public abstract partial class SortingOption : ModTexturedType, ILocalizedModType
```

Comment:
`ILocalizedModType` usage should not have been removed.  please use the existing code in MagicStorage/1.4.4 as a comparison